utility/block.go

```diff
@@ -133,10 +132,10 @@ func (u *UtilityContext) ApplyBlock() (string, error) {
 	// return the app hash (consensus module will get the validator set directly)
 	stateHash, err := u.Context.ComputeStateHash()
 	if err != nil {
-		log.Fatalf("Updating the app hash failed: %v. TODO: Look into roll-backing the entire commit...\n", err)
+		u.logger.Fatal().Err(err).Msg("Updating the app hash failed. TODO: Look into roll-backing the entire commit...")
 		return "", typesUtil.ErrAppHash(err)
 	}
-	log.Println("ApplyBlock - computed state hash:", stateHash)
+	u.logger.Info().Msgf("ApplyBlock - computed state hash: %s", stateHash)
```

Comment:
Should we use `.Str` here?

Reply:
https://github.com/pokt-network/pocket/pull/420#discussion_r1082884286